.github/workflows/issue-opened.yml

```
@@ -11,6 +11,8 @@ jobs:
   add-to-issue-triage:
     if: ${{ contains(github.event.issue.labels.*.name, 'bug :beetle:') }}
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
```

```suggestion
    permissions: {}
```
I believe this doesn't need any perms since it's using a GH app token, and it was already inheriting no perms.

---

`contents: read` is required for checkout to work